src/main/java/com/iexec/worker/compute/post/PostComputeService.java

```diff
@@ -233,25 +233,17 @@ public PostComputeResponse runTeePostCompute(TaskDescription taskDescription,
                 .build();
     }
 
-    private ReplicateStatusCause getExitCause(String chainTaskId, Integer exitCode) {
-        ReplicateStatusCause cause = null;
-        if (exitCode != null && exitCode != 0) {
-            switch (exitCode) {
-                case 1:
-                    // Use first cause from bulk processing for now
-                    cause = computeExitCauseService.getExitCausesAndPruneForGivenComputeStage(chainTaskId, ComputeStage.POST, POST_COMPUTE_FAILED_UNKNOWN_ISSUE).get(0);
-                    break;
-                case 2:
-                    cause = ReplicateStatusCause.POST_COMPUTE_EXIT_REPORTING_FAILED;
-                    break;
-                case 3:
-                    cause = ReplicateStatusCause.POST_COMPUTE_TASK_ID_MISSING;
-                    break;
-                default:
-                    break;
-            }
+    private List<ReplicateStatusCause> getExitCauses(String chainTaskId, Integer exitCode) {
+        if (exitCode == null || exitCode == 0) {
```

Comment:
are you sure about this ? How do you explain the difference with `PreComputeService` here ?